pages/common/git-push.md

```diff
@@ -3,17 +3,21 @@
 > Push commits to a remote repository.
 > More information: <https://git-scm.com/docs/git-push>.
 
-- Send local changes in the current branch to its remote counterpart:
+- Send local changes in the current branch to its default remote counterpart:
 
 `git push`
 
-- Send local changes in a given branch to its remote counterpart:
+- Send changes from a specific local branch to its remote counterpart:
 
 `git push {{remote_name}} {{local_branch}}`
 
-- Publish the current branch to a remote repository, setting the remote branch name:
+- Send changes from a specific local branch to its remote counterpart, and set the remote one as the default push/pull target of the local one:
 
-`git push {{remote_name}} -u {{remote_branch}}`
+`git push {{remote_name}} -u {{local_branch}}`
```

Comment:
Looking at the man page for `git push`, `-u` doesn't even take an argument:
```
git push [--all | --mirror | --tags] [--follow-tags] [--atomic] [-n | --d
ry-run] [--receive-pack=<git-receive-pack>]
                  [--repo=<repository>] [-f | --force] [-d | --delete] [--prune]
 [-v | --verbose]
                  [-u | --set-upstream] [-o <string> | --push-option=<string>]
                  [--[no-]signed|--signed=(true|false|if-asked)]
                  [--force-with-lease[=<refname>[:<expect>]] [--force-if-include
s]]
                  [--no-verify] [<repository> [<refspec>...]]

```

....so the ordering here is misleading. I suggest putting the `-u` either before or after `{{remote_name}}` and `{{local_branch}}`.

Reply:
I prefer before:
```suggestion
`git push --set-upstream {{remote_name}} {{local_branch}}`
```

Reply:
```suggestion
`git push -u {{remote_name}} {{local_branch}}`
```